src/main/java/eu/dissco/backend/repository/ElasticSearchRepository.java

```diff
@@ -52,6 +55,14 @@ public class ElasticSearchRepository {
   private final ObjectMapper mapper;
   private final ElasticSearchProperties properties;
 
+  private static Builder getTerm(String field, Builder t, boolean alphabetical) {
```

Comment:
This works a bit different. So this call get's a term provided with a prefix value and then returns everything in that term that matches the prefix. Normally this is sorted by the amount of times the prefix hits. However, for the name completion I think we should have an option to use alphabetical order. So this param changes the results to be sorted on alphabetical order. But for numbers this order will be different as it will sort it from 0, 1, 2 etc... so the order it uses depends on the type of the elasticsearch type. That is why I wanted to change it to a generic name "sort" as it basically sorts the results based on the sort in elastic. Does this make sense?

Reply:
yeah that makes sense! Maybe change the name of this variable to "sortAlphabetically" in all usages just for clarity?

Reply:
I changed it to sort as the sort for some elastic types might not be alphabetical